tests/test_tester_utils.py

```diff
@@ -76,3 +81,22 @@ def test_get_last_group(self):
         # Test the case of no match
         last_group = get_last_group(bucket, "TestCam", "20110101")
         self.assertEqual(last_group, int(20110101) * 100_000)
+
+    def test_exposure_id_hsc(self):
+        group = "2023011100026"
+        # Need a Butler registry to test ExposureIdInfo
+        with tempfile.TemporaryDirectory() as repo:
+            butler = butler_tests.makeTestRepo(repo)
+            HyperSuprimeCam().register(butler.registry)
+            for record in butler.registry.queryDimensionRecords("instrument", dataId={"instrument": "HSC"}):
```

Comment:
There should be only one, right?  Maybe check for that explicitly rather than iterating and taking only the last one.